src/Microsoft.VisualStudio.Editors/PropPages/ApplicationPropPageInternalBase.vb

```diff
@@ -130,7 +130,7 @@ Namespace Microsoft.VisualStudio.Editors.PropertyPages
                 ' TODO: Remove IsTargetFrameworksDefined check after issue #800 is resolved.
                 If TargetFrameworksDefined() = False And vsFrameworkMultiTargeting IsNot Nothing Then
 
-                    Dim supportedTargetFrameworksDescriptor As PropertyDescriptor = GetPropertyDescriptor("SupportedTargetFrameworks")
+                    Dim supportedTargetFrameworksDescriptor As PropertyDescriptor = GetPropertyDescriptor("SupportedTargetFrameworkAlias")
```

Comment:
This changes from plural to singular. Is that right?

Reply:
This should stay as it was, I was just able to test it.